kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -155,6 +155,12 @@ void setupWithBurrow(Map<String, Object> conf, Burrow burrow) {
     this.executor = MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workerThreadCount));
   }
 
+  // Validate an individual cluster's configuration. A cluster's configuration is valid if:
+  // a) Looking it up by name in Burrow returns a successful response (not 404)
+  // b)
```

Comment:
This looks weird, to me this should be `b) Bootstrap servers match those in burrow` and then you can have the i, ii below to describe behavior for each

Reply:
done